python/ray/dag/utils.py

```diff
@@ -37,10 +37,6 @@ def get_node_name(self, node: DAGNode):
         # serve components are not included in Ray Core.
         elif type(node).__name__ in ("DeploymentNode", "DeploymentFunctionNode"):
             node_name = node.get_deployment_name()
-        elif type(node).__name__ == "DeploymentMethodNode":
-            node_name = node.get_deployment_method_name()
-        elif type(node).__name__ == "DeploymentExecutorNode":
-            node_name = node._deployment_handle.deployment_name
         elif type(node).__name__ == "DeploymentMethodExecutorNode":
             node_name = node._deployment_method_name
         elif type(node).__name__ == "DeploymentFunctionExecutorNode":
```

Comment:
Do we still need the `DeploymentMethodExecutorNode` and the `DeploymentFunctionExecutorNode`?

Reply:
Nope, good catch!

Reply:
Sorry, misread: we don't need `DeploymentMethodExecutorNode`, but we still need `DeploymentFunctionExecutorNode`